src/SOS/SOS.Hosting/RuntimeWrapper.cs

```diff
@@ -383,7 +383,8 @@ private IntPtr CreateClrDataProcess(IntPtr dacHandle)
         private IntPtr CreateCorDebugProcess()
         {
             string dbiFilePath = _runtime.GetDbiFilePath();
-            string dacFilePath = _runtime.GetDacFilePath(out bool verifySignature);
+            // The DAC will be verified in the GetDacHandle call below. Ignore the verifySignature parameter here.
+            string dacFilePath = _runtime.GetDacFilePath(out bool _);
```

Comment:
Why have this call here? I'd move the trtace for DAC not found to where we do GetDacHandle and not do 2 separate calls. The error becomes clearer too. Also, why do we cache the dbi but not the dac handle?

Reply:
It is for the call to DBI's OpenVirtualProcessImpl2 on line 432 and on lines 458, 461.

GetDacHandle does cache the DAC handle.

Reply:
I moved it next to the GetDacHandle call.